src/include/wiredtiger.in

```diff
@@ -1897,6 +1907,25 @@ struct __wt_session {
 	int __F(timestamp_transaction)(WT_SESSION *session, const char *config);
 	/*! @} */
 
+	/*!
+	 * Set the commit timestamp on a transaction numerically.  Prefer this over @ref
+	 * timestamp_transaction when the string parsing done in that method becomes a bottleneck.
+	 *
+	 * The WT_SESSION.timestamp_transaction_numeric method can only be used at snapshot
+	 * isolation.
+	 *
+	 * @requires_transaction
+	 *
+	 * @param session the session handle
+	 * @param which the timestamp you wish to set. See @ref WT_TX_TXN_TYPE for available
+	 * options.
+	 * Durable timestamp should be set only after setting the commit timestamp.
+	 * @param ts the timestamp. Zero means none.
+	 * @errors
+	 */
+	int __F(timestamp_transaction_numeric)(WT_SESSION *session, WT_TS_TXN_TYPE which,
```

Comment:
I prefer `timestamp_transaction_int`, or even `xxx_uint`, over `xxx_numeric`. Ignore me if the consensus is for `xxx_numeric`, but I do prefer the others.

Reply:
That's fair. I preferred `numeric` since `int` made me think "internal" which could've been confusing. I like `uint` though, does anyone else here have an opinion on this?

Reply:
I would prefer `uint` over `numeric`. I can understand why `int` could be mis-interpreted as internal.

Reply:
Changed in 3cfe81c, thanks for the feedback.